tests/lib/nested.sh

```diff
@@ -1204,13 +1204,20 @@ nested_start_core_vm_unit() {
     PARAM_RTC="${NESTED_PARAM_RTC:-}"
     PARAM_EXTRA="${NESTED_PARAM_EXTRA:-}"
 
+    local EXPECT_PASSPHRASE
+    EXPECT_PASSPHRASE=${NESTED_EXPECT_PASSPHRASE:-}
```

Comment:
It seems that the practice is insted to have a statement like `: "${NESTED_EXPECT_PASSPHRASE:=}"` at the bgining of nested.sh